app/src/main/java/com/brainwallet/ui/BrainwalletActivity.kt

```diff
@@ -158,6 +170,24 @@ class BrainwalletActivity : BRActivity() {
             .launchIn(lifecycleScope)
     }
 
+
+    /**
+     * legacy logic, when the pin/passcode empty (not set)
+     * then should go to setpasscode
+     */
```

Comment:
This is actually an elegant solution @andhikayuana .  Even though the logic was supposed to do it there are too many screens in the step